test/generators/html/Type.html

```diff
@@ -89,11 +95,17 @@ <h1>Module <code><span>Type</span></code></h1>
       <span><span class="keyword">type</span> labeled_higher_order</span>
       <span> = 
        <span>
-        <span>(<span>l:int <span class="arrow">&#45;&gt;</span></span> int)
+        <span>(
```

Comment:
Not related to this PR but this reminds me that the `--indent` option breaks the formatting: https://github.com/ocsigen/tyxml/issues/288